build/Dockerfile

```diff
@@ -301,11 +301,13 @@ RUN CGO_ENABLED=0 GOFLAGS='-mod=vendor' \
 FROM files AS container
 COPY --chown=nginx:0 --from=builder /nginx-ingress /
 
+STOPSIGNAL SIGTERM
 USER nginx
 
 
 ############################################# Create image with nginx-ingress built locally #############################################
 FROM files AS local
 COPY --chown=nginx:0 nginx-ingress /
 
+STOPSIGNAL SIGTERM
```

Comment:
you can just have the `STOPSIGNAL` once if you put it in `files` like on line 288